var/spack/repos/builtin/packages/npm/package.py

```diff
@@ -25,6 +25,7 @@ class Npm(Package):
     version("3.10.5", sha256="ff019769e186152098841c1fa6325e5a79f7903a45f13bd0046a4dc8e63f845f")
 
     depends_on("node-js", type=("build", "run"))
+    depends_on("libvips")
```

Comment:
Without this I see:
```
../src/common.cc:25:10: fatal error: 'vips/vips8' file not found
#include <vips/vips8>
         ^~~~~~~~~~~~
```
With this, I get a bit further in the build, but it still fails:
```
FATAL ERROR: wasm code commit Allocation failed - process out of memory
...
bash: line 1: 83387 Abort trap: 6           node ../bin/npm-cli.js run build:static echo 1>&2
```
Not sure what's wrong, but this version is also super old. Would love to update npm, but newer versions don't have a configure script.